src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -71,20 +85,23 @@ export class ZaakCreateComponent implements OnInit {
 
         const titel = new HeadingFormFieldBuilder().id('aanmakenZaak').label('actie.zaak.aanmaken').level('1').build();
 
-        const tussenTitel = new HeadingFormFieldBuilder().id('overigegegevens').label('gegevens.overig').level('2').build();
+        const toekennenGegevensTitel = new HeadingFormFieldBuilder().id('toekennengegevens').label('gegevens.toekennen').level('2').build();
+
+        const overigeGegevensTitel = new HeadingFormFieldBuilder().id('overigegegevens').label('gegevens.overig').level('2').build();
 
         const zaaktype = new SelectFormFieldBuilder().id('zaaktype').label('zaaktype')
                                                      .validators(Validators.required)
                                                      .optionLabel('omschrijving')
                                                      .options(this.zakenService.listZaaktypes())
                                                      .build();
 
+        this.subscription$ = zaaktype.formControl.valueChanges.subscribe(v => this.zaaktypeGeselecteerd(v));
+
         const startdatum = new DateFormFieldBuilder().id('startdatum').label('startdatum')
                                                      .value(moment()).validators(Validators.required)
                                                      .build();
 
-        const registratiedatum = new DateFormFieldBuilder().id('registratiedatum').label('registratiedatum')
-                                                           .value(moment()).build();
+        this.medewerkerGroepFormField = this.getMedewerkerGroupFormField();
```

Comment:
Ik zou die method nog inlinen, want het is nu de enige waarvoor je ergens anders moet kijken om te zien wat de control kan.

Reply:
Ik heb hem hier niet inline gezet omdat hij ook wordt hergebruikt op regel 204 (in `zaaktypeGeselecteerd`)